analysis/model/src/main/kotlin/de/maibornwolff/codecharta/serialization/OutputFileHandler.kt

```diff
@@ -9,13 +9,16 @@ object OutputFileHandler {
         return if (outputFilePath.isNullOrBlank()) {
             fallbackOutputStream
         } else {
-            FileOutputStream(File(checkAndFixFileExtension(outputFilePath, compressed)))
+            FileOutputStream(File(checkAndFixFileExtensionJson(outputFilePath, compressed)))
         }
     }
 
-    fun checkAndFixFileExtension(outputName: String, compressed: Boolean): String =
+    fun checkAndFixFileExtensionJson(outputName: String, compressed: Boolean): String =
```

Comment:
I think it would be better to have a new parameter for the file extension and to keep the old method name.
Thus, we could handle the different cases within the OutputFileHandler, so that you do not have to take care on your own in the outside world (tell dont ask principle).

Another advantage would be that a new file extension could be easily added in one place:

```
fixFileExtension(fileName, extension) {
    fixFileExtensionJson(fileName, extension);
    fixFileExtensionCsv(fileName, extension);
}

fixFileExtensionJson(fileName, extension) {
    if (extension == json) {
       // ...
    }
}

// ....
```